test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,97 @@
+import {
+  generateUser,
+  resetHabiticaDB,
+  generateGroup,
+} from '../../../../helpers/api-v3-integration.helper';
+
+describe('GET /groups', () => {
+  let user;
+  const NUMBER_OF_PUBLIC_GUILDS = 3;
+  const NUMBER_OF_USERS_PRIVATE_GUILDS = 1;
+  const NUMBER_OF_GUILDS_USER_CAN_VIEW = 6;
+
+  before(async () => {
+    await resetHabiticaDB();
+
+    let leader = await generateUser({ balance: 10 });
+    user = await generateUser({balance: 4});
+
+    let publicGuildUserIsMemberOf = await generateGroup(leader, {
```

Comment:
Probably because it's not part of the interface to add an existing user in the `createAndPopulateGroup` helper. Should probably add that functionality.